language-design.md

```diff
@@ -17,7 +17,17 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+"...Perl, a major programming language used by untold zillions of developers, is no more intuitive to novices than a language with a randomly generated syntax." [Pavlus, 2012]
 
+I definitely disagreed with Pavlus' article, especially the claim quoted above. From my experience on the OCM project, I know that it can be hard to do studies on languages. Even though OCM was better from the standpoint of someone who was actually acquainted with parallel programming, it was hard to use novices to show this. User studies over the years have been problematic and it's very hard to teach people parallel programming over the course of half an hour or so. There are also a lot of problems with the idea of using novices to judge a language when they make up a very small portion of the userbase. Not all languages are designed to be easy to learn or to be very user friendly. In fact, I think it would be counterintuitive for most programming languages to have ease of use for novices as a primary concern. User-friendly languages will have to be more verbose than regular languages and will make doing just basic tasks a chore. In the example given in the article, a for loop in Quorum will take twice as many lines as C and about double to triple the number of characters. While languages like Quorum or Scratch may be more accessible to novices, they would likely hinder the productivity of most programmers in the long run. It's also worth noting that the paper only used a sample size of 18 people whose average age was 21 [[Stefik et al., 2011, p 5](http://ecs.victoria.ac.nz/foswiki/pub/Events/PLATEAU/2011Program/plateau2011-stefik.pdf)]. Even if the results were statistically significant, 18 people is a pretty small sample size. Their sample also probably don't represent many of the people learning to code, especially since many coding novices are significantly younger than 21.
```

Comment:
Interesting point. I think there's a small disconnect, however, between what the quote is saying and the larger point that you're addressing. The quote is simply saying that Perl is no more intuitive, not addressing the actual meaning of this result. From my experience with Perl, I don't doubt this, in the sense that looking at a Perl program with zero coding knowledge would just seem like random characters. I do recognize the difficulty of doing studies like this with the points you make, but the claim itself seems fairly simple. Overall though, i do agree with you in the sense that this inherent difficulty with Perl (and languages in general) is not necessarily a problem.